third-party/sysdeps/linux/libdrm/inline_amdgpu_ids.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Reads the data/amdgpu.ids file and replaces the amdgpu/amdgpu_asic_id.c file
+# with a fully inlined table that does not need to consult a fixed place on
+# the filesystem.
+
+SOURCE_DIR="${1:?Expected source directory}"
+
+generate() {
+  echo '
+#include <ctype.h>
+#include <stdint.h>
+#include <string.h>
+#include "amdgpu_drm.h"
+#include "amdgpu_internal.h"
+struct inline_amdgpu_id {
+  uint32_t did;
+  uint32_t rid;
+  const char *name;
+};
+'
+  # Output an inline table.
+  echo "static struct inline_amdgpu_id inline_amdgpu_ids[] = {"
```

Comment:
It doesn't makes a difference here but as the above `echo` uses single quotes, one could do here (and at some of the echo calls below) as well.

Reply:
Done